internal/datastore/mongo_store.go

```diff
@@ -0,0 +1,97 @@
+package datastore
+
+import (
+	"context"
+	"log"
+	"time"
+
+	"github.com/SimplQ/simplQ-golang/internal/models"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+// A Structure with Collections frequently used and a pointer to the client
+type MongoDB struct {
+    Client *mongo.Client
+    Queue *mongo.Collection
+    Token *mongo.Collection
+}
+
+var mongodb MongoDB
+
+func ConnectMongoDB(uri string) *MongoDB {
+    log.Println("Connection to MongoDB...")
+
+    client, err := mongo.Connect(context.TODO(), options.Client().ApplyURI(uri))
+    
+    if err != nil {
+        log.Fatal(err)
+    }
+
+    mongodb = MongoDB {
```

Comment:
Why did we store to a global variable here? Can we avoid it?

Reply:
oh thanks for pointing that out
I accidentally used the same variable as the global
Will update